kratos/python_scripts/set_initial_state_process.py

```diff
@@ -137,16 +154,16 @@ def ExecuteInitializeSolutionStep(self):
             nr_cols = len(self.deformation_functions[0])
             for r in range(nr_rows):
                 for c in range(nr_cols):
-                    self.imposed_deformation_gradient[r, c] =  \
-                        self.deformation_functions[r][c].CallFunction(
-                            0, 0, 0, current_time, 0, 0, 0
-                        )
+                    self.imposed_deformation_gradient[
+                        r, c
+                    ] = self.deformation_functions[r][c].CallFunction(
+                        0, 0, 0, current_time, 0, 0, 0
+                    )
```

Comment:
Minor: I think that in this case multiline doesn't enhance code readability (feel free to dismiss).

Reply:
Yeah. It was my formater (black). Fixed